src/lib/network/install_inf_convertor.rb

```diff
@@ -47,11 +47,6 @@ def write_netconfig
       write_global_netconfig
     end
 
-    # Reports if user asked for using biosdevname pernament device names
-    def AllowUdevModify
-      /biosdevname=1/ !~ InstallInf["Cmdline"]
```

Comment:
It is ok to remove it as it is not used anymore. But maybe we should add some note in the udev rules writers. In the past this check was part of the LanItems writer and the udev rules were not written in case that the use of predictable network interface names was choosen.

Reply:
I've dropped a comment into writer. Nobody complained so far, so let's see whether it is going to be needed in the future.